src/s3-update.ts

```diff
@@ -29,11 +30,11 @@ export async function s3Update(options: S3UpdateOptions): Promise<void> {
   // However, branches are not intended for production use, so the occasional broken
   // branch does not seem worth fixing.
 
-  const { deployPath, version, branch, bucket, prefix, topBranchesJSON, localFolder, noPrefix} = options;
+  const { deployPath, version, branch, bucket, prefix, topBranchesJSON, localFolder, noPrefix, maxAge} = options;
 
   const topLevelS3Url = noPrefix ? `s3://${bucket}` : `s3://${bucket}/${prefix}`;
   const deployS3Url = `${topLevelS3Url}/${deployPath}`;
-  const maxAgeSecs = version ?  MAX_AGE_VERSION_SECS : MAX_AGE_BRANCH_SECS;
+  const maxAgeSecs = maxAge || (version ?  MAX_AGE_VERSION_SECS : MAX_AGE_BRANCH_SECS);
```

Comment:
If someone wants to set the maxAge to `0` for the version folder this test won't allow that.

Reply:
A good use case for `??`

Reply:
Fixed in https://github.com/concord-consortium/s3-deploy-action/pull/396/commits/e77d78b685c582080cb88f44c4eb0adbf4e0c24a using `??`.